packages/story-editor/src/app/highlights/quickActions/useQuickActions.js

```diff
@@ -331,6 +340,7 @@ const useQuickActions = () => {
     muteAudio: actions.muteAudio,
     unMuteAudio: actions.unMuteAudio,
     startTrim: actions.startTrim,
+    setVideoEffect: actions.setVideoEffect,
   }));
 
   const enableMediaRecordingTrimming = useFeature('recordingTrimming');
```

Comment:
<!-- ideas that popped up from reviewing -->
**💡 thought:**
Not related to this PR directly but should these actions be disabled during recording? Currently it seems that it's possible to click on it during recording which results in the recording just stopping. Thoughts?

Reply:
Hm, okay, I'm now trying again and seeing these are disabled generally. Unfortunately I don't know how to reproduce it but somehow I ended up (twice) in a situation where I could click on the Disable Background Blur quick-action while recording -> recording stopped -> when I opened the recorder again, I saw this with neither of the buttons (Pause/Stop) doing anything:
![Screenshot 2022-08-19 at 13 36 25](https://user-images.githubusercontent.com/3294597/185601436-801f4c06-b22f-4828-83e1-c276a56587e4.png)

Reply:
OK, I tried again and the following seems to break it eventually:
- Open the recorder
- Enable background blur
- Start recording
- Wait for a couple of seconds -- when testing, the quick-actions became enabled when the blur option was turned on.
- Now disable the blur while recording. And now the video stops recording. 
- Click X to close the recorder tool.

When I repeat the process a few times, the following occurs:
- If the video stopped recording because of using quick-actions during recording, and then when opening the recorder tool from the Library again, it starts recording right away when opening.
- After a few times of doing this, the video starts recording but doesn't really show the video and the Pause/Stop buttons don't do anything (as above on the screenshot).

Reply:
I thought it would be just a flag change (incorrect status/inputStatus/isReady), but it turned out that I was trying to fix something that was impossible to fix with the current version of the library. It was not possible to use the custom media recorder more than once per instance. Luckily, when I was about to go crazy I checked the library repo and saw [this one-liner](https://github.com/wmik/use-media-recorder/commit/94d06776129f7efdd9992785fcf2e2e400256a4d) 🪄 

_Note to self: always check if you are using the latest version of the library._